app/controllers/.fuse_hidden00002d9a0000001d

```diff
@@ -0,0 +1,17 @@
+class ApplicationController < ActionController::Base
```

Comment:
Can't even pull that hidden file back. I've added it to .gitignore (just in case)